core/src/main/kotlin/org/evomaster/core/problem/graphql/GraphQLActionBuilder.kt

```diff
@@ -1,105 +1,689 @@
 package org.evomaster.core.problem.graphql
 
-import ch.qos.logback.classic.db.names.TableName
 import com.google.gson.Gson
-import io.swagger.v3.oas.models.OpenAPI
+import org.evomaster.core.logging.LoggingUtil
+import org.evomaster.core.problem.graphql.param.GQInputParam
+import org.evomaster.core.problem.graphql.param.GQReturnParam
 import org.evomaster.core.problem.graphql.schema.SchemaObj
+import org.evomaster.core.problem.graphql.schema.__TypeKind
+import org.evomaster.core.problem.rest.param.Param
 import org.evomaster.core.search.Action
+import org.evomaster.core.search.gene.*
 import java.util.concurrent.atomic.AtomicInteger
-
+import java.util.*
+import org.slf4j.Logger
+import org.slf4j.LoggerFactory
 
 object GraphQLActionBuilder {
 
+    private val log: Logger = LoggerFactory.getLogger(GraphQLActionBuilder::class.java)
     private val idGenerator = AtomicInteger()
+    var tables: MutableList<Table> = mutableListOf()
+    var argsTables: MutableList<Table> = mutableListOf()
+    var tempArgsTables: MutableList<Table> = mutableListOf()
```

Comment:
why are these fields public? (recall that in Kotlin, the default scope is public)